en/docs/concepts/compliance/gdpr.md

```diff
@@ -0,0 +1,229 @@
+# General Data Protection Regulation
+
+## About GDPR compliance
+
+The General Data Protection Regulation (GDPR) is a legal framework formalized by the European Union (EU) in 2016. This regulation came into effect from 25, May 2018, and affects any organization that processes Personally Identifiable Information (PII) of individuals who live in Europe. Organizations that fail to demonstrate GDPR compliance are
+subjected to financial penalties.
+
+WSO2 Identity Server (WSO2 IS) is designed based on privacy best practices, and is fully compliant with GDPR. GDPR compliance in IAM and API security spaces can be completely fulfilled with WSO2 IS. This section outlines the major GDPR-related features that WSO2 IS supports, and also provides necessary references to specific concepts within the section.
+
+!!! Info
+    To understand GDPR more elaborately, take a look at this tutorial series on *Creating a Winning GDPR Strategy*.
+
+    -   Part 1 - [Introduction to
+        GDPR](https://wso2.com/library/article/2017/12/introduction-to-gdpr/)
+
+    -   Part 2 - [7 Steps for GDPR
+    Compliance](https://wso2.com/library/article/2017/12/7-steps-for-gdpr-compliance/)
+
+    -   Part 3 - [Identity and Access Management to the
+        Rescue](https://wso2.com/library/article/2018/2/identity-and-access-management-to-the-rescue/)
+
+    -   Part 4 - [GDPR Compliant Consent
+        Design](https://wso2.com/library/articles/2018/03/creating-a-winning-gdpr-strategypart-4-gdpr-compliant-consent-design/)
+
+    For more resources on GDPR, see the white papers, case studies, solution
+    briefs, webinars, and talks published on our [WSO2 GDPR homepage](https://wso2.com/solutions/regulatory-compliance/gdpr/). See the original GDPR legal text
+    [here](http://eur-lex.europa.eu/legal-content/en/TXT/?uri=CELEX%3A32016R0679).
+
+## Privacy by design and privacy by default
+
```

Comment:
the content doesn't explain what privacy by design and privacy by default is and the difference between them. 
check: https://auth0.com/docs/compliance/gdpr#privacy-by-design-and-privacy-by-default